website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -12,99 +12,177 @@ These instructions are not intended for general audiences at this time.
-dbt --help
+tar -xf dbt_0.29.9_linux_amd64.tar.gz
+./dbt --version
 ```
 
-### Manually install (Windows and Linux)
+:::info
 
-1. Download the latest release for your platform from [GitHub](https://github.com/dbt-labs/dbt-cli/releases).
-2. Add the `dbt` executable to your path.
-3. Move to a directory with a dbt project, and create a `dbt_cloud.yml` file containing your `project-id` from dbt Cloud.
-4. Invoke `dbt --help` from your terminal to see a list of supported commands.
+Advanced users can configure multiple projects to use the same Cloud CLI executable by adding it to their PATH environment variable in their shell profile.
 
-#### Updating your dbt Cloud installation (Windows + Linux)
+:::
 
-Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-only) and replace the existing `dbt` executable with the new one. You should not have to go through the security steps again.
+3. Verify the installation by running `./dbt --help` from the command line. If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
 
-## Setting up the CLI
+* You no longer need to use the `dbt deps` command. Previously, you had to run that command.
 
-The following instructions are for setting up the dbt Cloud CLI. 
+</TabItem>
 
-1. Ensure that you have created a project in [dbt Cloud](https://cloud.getdbt.com/).
+</Tabs>
 
-2. Ensure that your personal [development credentials](https://cloud.getdbt.com/settings/profile/credentials) are set on the project.
+## Update dbt Cloud CLI
 
-3. Navigate to [your profile](https://cloud.getdbt.com/settings/profile) and enable the **Beta** flag under **Experimental Features.**
+The following instructions explain how to update the dbt CLoud CLI to the latest version depending on your operating system. 
 
-4. Create an environment variable with your [dbt Cloud API key](https://cloud.getdbt.com/settings/profile#api-access):
+#### Update with Homebrew (MacOS)
 
-```bash
-vi ~/.zshrc
+To update the dbt Cloud CLI, run `brew upgrade dbt-cloud-cli`. 
+#### Update manually (Windows and Linux)
 
-# dbt Cloud CLI
-export DBT_CLOUD_API_KEY="1234" # Replace "1234" with your API key   
-```
+To update, follow the same process explained in [Install manually (Windows)](#install-manually-windows) and replace the existing `dbt.exe` executable with the new one.
+
+
+## Configure the dbt Cloud CLI
 
-5. Load the new environment variable. Note: You may need to reactivate your Python virtual environment after sourcing your shell's dot file. Alternatively, restart your shell instead of sourcing the shell's dot file
+After installation, you can configure the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, you can execute the following commands to compile a project using dbt Cloud:
 
 ```bash
-source ~/.zshrc
+dbt compile
 ```
```

Comment:
```suggestion
```
* You no longer need to use the `dbt deps` command. Previously, you had to run that command.
```